plugins/faucet/plugin.go

```diff
@@ -38,16 +36,12 @@ const (
 
 var (
 	// Plugin is the "plugin" instance of the faucet application.
-	Plugin                   *node.Plugin
-	_faucet                  *StateManager
-	powVerifier              = pow.New()
-	fundingWorkerPool        *workerpool.NonBlockingQueuedWorkerPool
-	fundingWorkerCount       = runtime.GOMAXPROCS(0)
-	fundingWorkerQueueSize   = 500
-	preparingWorkerPool      *workerpool.NonBlockingQueuedWorkerPool
-	preparingWorkerCount     = runtime.GOMAXPROCS(0)
-	preparingWorkerQueueSize = MaxFaucetOutputsCount + 1
-	targetPoWDifficulty      int
+	Plugin              *node.Plugin
+	_faucet             *Faucet
+	powVerifier         = pow.New()
+	requestChanSize     = 100
+	requestChan         = make(chan *faucet.Payload, requestChanSize)
+	targetPoWDifficulty int
 	// blacklist makes sure that an address might only request tokens once.
 	blacklist         *orderedmap.OrderedMap[string, bool]
```

Comment:
Remove the whole blacklist logic along with those fields as we don't need it and want to simplify as much as possilbe